packages/db/fauna/schema.graphql

```diff
@@ -215,6 +215,36 @@ type PinSyncRequest {
   created: Time!
 }
 
+"""
+Details of the backups created for an upload.
+"""
+type Backup {
+  """
+  Upload that originated the backup
+  """
+  upload: Upload! @relation
+
+  """
+  Backup key.
+  """
+  key: String!
+
+  """
+  Backup bucket name.
+  """
+  name: String!
```

Comment:
`name` `is ambigious here, it should be `bucketName`.

Reply:
I named backup and name here taking into consideration that we are using S3, but it could be any other solution. I would prefer to use agnostic names that are not tied to the service we use. What do you think?